commands/daemon.go

```diff
@@ -133,7 +133,8 @@ func runAPIAndWait(ctx context.Context, nd *node.Node, config *config.Config, re
 		ctx:          context.Background(),
 		api:          api,
 		porcelainAPI: nd.PorcelainAPI,
-		blockpAPI:    nd.BlockAPI,
+		blockAPI:     nd.BlockAPI,
```

Comment:
I know this is after the fact but imo blockAPI should be blockMiningAPI.

Reply:
Agree

Reply:
Naming is hard.  I struggled between FooMining, FooProtocol and between keeping names short a la go convention.  I'll put it as a todo in the cleanup issue I just filed.